### ISSUE TITLE
passing lambdas to `use:` hooks causes warnings

### ISSUE THREAD

**Issue by aminya** [solidjs-community/eslint-plugin-solid#85]
**Describe the bug**
passing lambdas to `use:` hooks causes warnings

```
This function should be passed to a tracked scope (like createEffect) or an event handler because it contains reactivity, or else changes will be ignored.eslint[solid/reactivity](https://github.com/solidjs-community/eslint-plugin-solid/blob/main/docs/reactivity.md)
```

**To Reproduce**

```jsx
const someHook = createSomeHook();
someHook;

<div
        use:someHook={() => {
            // do something
        }}
      >
```

**Expected behavior**

**Screenshots**

**Environment (please complete the following information):**

- OS: [e.g. Mac OS 11, Windows 10]
- Node version (`node --version`): 18.14.2
- `eslint-plugin-solid` version (`npm list eslint-plugin-solid`/`yarn why eslint-plugin-solid`): 0.11
- `eslint` version (`npm list eslint`/`yarn why eslint`): 8.4.1

**Additional context**

- [ ] I would be willing to contribute a PR to fix this issue
